source/system-administrators/upgrade/upgrade-notes-for-re-encrypting-values.rst

```diff
@@ -83,7 +85,19 @@ Manually Re-encrypt Values
 
       .. code-block:: bash
 
-         java -jar crafter-commons-utilities-3.1.9-enctool.jar -d VkHxBsaSZ9DXrXk52uK9And+CEZlqiy7Wb23GxBFOSXD6KBOCh1ojp8fUw7w11IxpxBipiI4HsSg3cdl9TgTQg== -p klanFogyetkonjo -s S25pT2RkeWk=
+         $ java -jar crafter-commons-utilities-3.1.10-enctool.jar -e mySup3rsecret#@hello -p klanFogyetkonjo -s Sdf25pT2RkeWk=
+         Cipher text (in Base 64): VkHxBsaSZ9DXrXk52uK9And+CEZlqiy7Wb23GxBFOSXD6KBOCh1ojp8fUw7w11IxpxBipiI4HsSg3cdl9TgTQg==
```

Comment:
The new value can't be the same, otherwise there is something wrong in the process. The new value should look something like this: `CCE-V1#QVEemADrH2yuR6vAMZfL64e8Lp0JD1ihQbbFwNGReyrWM21HfX31tdPEp6jjbzrk`